pyproject.toml

```diff
@@ -42,7 +42,6 @@ path = "src/chronos/__about__.py"
 extras = [
   "boto3>=1.10,<2",
   "peft>=0.13.0,<0.18",
-  "fev>=0.6.1",
```

Comment:
How about we keep all the `extras` here, but change the dependencies in AG-TS to
```
chronos-forecasting>=...
peft>=...,<...
```
We do something similar with `torch`, where we don't depend on `gluonts[torch]`, but rather install `torch` as a dependency in AG based on our version range.